research-software-directory/RSD-as-a-service
# WIP: 306 move is_featured from software table to related tables

# Move is featured

Added mention and contributor count to software and related software cards and applied the order

TODO: implement data-migration is_featured value for escience center

Closes #306

Changes proposed in this pull request:
* Is featured switch removed from software
* is_featured prop removed from software table and moved to related tables
* Mention and contributor count added to software card instead of Featured text
* Software ordered on number of mentions

How to test:
* docker-compose down --volumes
* docker-compose build
* docker-compose up
* run data migration, it should work
* software should be ordered by mention count, if mention count > 0 the card will be colored blue (as featured in the past) on the main overview page
* visit escience center company, is_featued should be available there

PR Checklist:
